src/options.h

```diff
@@ -1884,11 +1884,18 @@ donot_indent_func_def_close_paren;
 ///////////////////////////////////////////////////////////////////////////////
 //BEGIN Newline adding and removing options
 
-// Whether to collapse empty blocks between '{' and '}'.
-// If true, overrides nl_inside_empty_func
+// Whether to collapse empty blocks between '{' and '}' except for functions.
```

Comment:
The description here is fine, but it needs to be updated in various other files so that it is aligned everywhere (see .cfg and doc files)

Reply:
Ah, forgot. Thanks for the reminder.

Is there a way to automate this perhaps? The number of files that need updating are quite a lot and it's easy to forget to update all of them.

Reply:
Better yet, is there a way to not need multiple identical files that are scattered around the codebase?

Reply:
That is a good point, it is something to look at on my TODO list as well.
Basically config.txt and defaults.cfg in the htdocs folder are the same as defaults.cfg in etc folder. 
universalindent.cfg in the test folder should be (IMO) the same as uigui_uncrustify.ini in etc, but currently it isn't. It is autogenerated, so there may be something to fix in the generation process..
The question is whether we need all those files or if some could be removed. Then if some duplicates are needed, we could look at automate the process, so we only update etc/defaults.cfg and the other files are autogenerated from it.
I need to check with @guy-maurel about the past history of these files to know more.